### PR TITLE
Limelight Digital Bid Adapter: Prevent duplicate iframe and pixel syncs

### PR DESCRIPTION
**Pull request by apykhteyev**
## Type of change
- [ ] Feature

## Description of change
Respond with user sync pixels for different customers of limelight digital

### CODE REVIEW

**Comment by apykhteyev** [comment]
Doesn't seems that I could break something in Criteo test here

File: modules/limelightDigitalBidAdapter.js [hunk 1]
```diff
@@ -94,23 +94,22 @@ export const spec = {
...
-          url: url
-        }
-      });
-    const pixelSyncUrls = !syncOptions.pixelEnabled ? [] : syncs.map(sync => sync.pixel).filter(Boolean)
-      .filter(uniques).map(url => {
-        return {
-          type: 'image',
-          url: url
+    let iframeSyncs = [];
+    let imageSyncs = [];
```
Comment (osazos) [inline modules/limelightDigitalBidAdapter.js:98]: Should be `const` as you don't reassign them

File: modules/limelightDigitalBidAdapter.js [hunk 2]
```diff
@@ -94,23 +94,22 @@ export const spec = {
...
+    for (let i = 0; i < serverResponses.length; i++) {
+      const serverResponseHeaders = serverResponses[i].headers;
+      const imgSync = (serverResponseHeaders != null && syncOptions.pixelEnabled) ? serverResponseHeaders.get('X-PLL-UserSync-Image') : null
+      const iframeSync = (serverResponseHeaders != null && syncOptions.iframeEnabled) ? serverResponseHeaders.get('X-PLL-UserSync-Iframe') : null
+      if (iframeSync != null) {
+        iframeSyncs.push(iframeSync)
+      } else {
+        if (imgSync != null) {
```
Comment (osazos) [inline modules/limelightDigitalBidAdapter.js:106]: You should use `else if` for easy reading

**Review by osazos — COMMENTED** [review]
Seems ok for me, but I get a server call error (Cors) when I try your adapter locally with the params found in your limelightDigitalBidAdapter.md file.

If it is possible, can you update your .md file to have valid params?

**Comment by apykhteyev** [comment]
> Seems ok for me, but I get a server call error (Cors) when I try your adapter locally with the params found in your limelightDigitalBidAdapter.md file.
> 
> If it is possible, can you update your .md file to have valid params?

Please check it now

**Review by osazos — APPROVED** [review]
LGTM

**Pull request merged**